pkg/normalizedtranscriptions/sipa.json

```diff
@@ -1,10 +1,12 @@
 [
   {
-    "conditions": {"aspiration": "aspirated", "voicing": "voiced"},
+    "conditions": {"aspiration": "aspirated", "phonation": "voiced"},
     "modifications": {"aspiration": null, "breathiness": "breathy"}
```

Comment:
There is a comma missing AFTER the } (think of a list of things in brackets, like `{}, {}, {}`.